lab_5_scrapper/scrapper.py

```diff
@@ -204,13 +450,78 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    base_path.mkdir(parents=True, exist_ok=True)
+
+    for file in base_path.iterdir():
+        file.unlink(missing_ok=True)
+
+
+def clear_recursive_crawler(base_path: Union[pathlib.Path, str]) -> None:
+    """
+        Create ASSETS_PATH folder if no created and create a file for crawled links.
+
+        Args:
+            base_path (Union[pathlib.Path, str]): Path where articles stores
+        """
+    prepare_environment(base_path)
+
+    crawled_path = base_path.parent / 'already_crawled_urls.txt'
+    with open(crawled_path, 'w', encoding='utf-8'):
+        pass
 
 
 def main() -> None:
     """
     Entrypoint for scrapper module.
     """
+    configuration = Config(path_to_config=constants.CRAWLER_CONFIG_PATH)
+
+    prepare_environment(base_path=constants.ASSETS_PATH)
+
+    crawler = Crawler(config=configuration)
+    crawler.find_articles()
+
+    for index, url in enumerate(crawler.urls):
+        parser = HTMLParser(full_url=url, article_id=index + 1, config=configuration)
+        article = parser.parse()
+        if isinstance(article, Article):
+            to_raw(article)
+            to_meta(article)
+    print("It's done!")
+
+
+def main_recursive() -> None:
+    """
+    Entrypoint for recursive scrapper module.
+    """
+    configuration = Config(path_to_config=constants.CRAWLER_CONFIG_PATH)
+
+    clear_recursive_crawler(constants.ASSETS_PATH)
```

Comment:
clearing should be just possible with removing intermediate state of the crawler